crates/bevy_reflect/bevy_reflect_derive/src/fq_std.rs

```diff
@@ -0,0 +1,77 @@
+use proc_macro2::TokenStream;
+use quote::{quote, ToTokens};
+
+// This module contains unit structs that should be used inside `quote!` and `spanned_quote!` using the variable interpolation syntax in place of their equivalent structs and traits present in `std`.
```

Comment:
Normally module doc comments are made using `//!` at the very top of the file (above imports). If we include these, could we move them to the top to be consistent with that format?

Reply:
I have moved them to the top and used `//!` for the docs.
Initially, I had thought that using `//!` would make this module visible in the user-facing documentation of Bevy but later realized that won't happen.